src/components/Layout/Header/TxWindow/TxWindow.tsx

```diff
@@ -54,9 +54,10 @@ const TxsByStatus: React.FC<TxsByStatusProps> = ({ txStatus, limit }) => {
   }, [limit, txIds])
 
   if (limitTxIds.length === 0) {
+    const translatedStatus = translate(`transactionRow.${txStatus.toLowerCase()}`)
```

Comment:
Translations are missing `TxStatus.SUCCESS` though it's not needed as success Txs shouldn't appear in the Tx window ✅